gremlin-javascript/src/main/javascript/gremlin-javascript/package-lock.json

```diff
@@ -4,6 +4,16 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@ampproject/remapping": {
```

Comment:
Wondering why the package-lock file is adding a large chunk of code again, is it the node version?

Reply:
installed with same node and npm version, the changes refer to the dependencies and sub-dependencies of the `nyc` dependency that is added in this PR

Reply:
added 100 dependencies, but it's ok for node world %)